cosmos/pacific.json

```diff
@@ -0,0 +1,119 @@
+{
+  "rpc": "https://rpc.wallet.pacific-1.sei.io",
```

Comment:
Let's add a public provider like Rhino here? We don't want people using this url except wallet providers.